_includes/posts/header.html

```diff
@@ -13,7 +13,7 @@
                         {% if page.header %}
                         <img src="{{ page.header }}" class="img-fluid">
                         {% endif %}
-                        <h2 class="display-4 slim pt-4">{{ page.title }}</h2>
+                        <h1 class="display-4 slim pt-4">{{ page.title }}</h1>
```

Comment:
@htahir1 this is the h1 / h2 fix, but I have a feeling it'll change the look of the blog pages to something we don't like.

Reply:
@strickvl lets try it and revert it if its bad